sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/HoldOptions.java

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.callautomation.models;
+
+import com.azure.communication.common.CommunicationIdentifier;
+
+/**
+ * Options for the Hold operation.
+ */
+public class HoldOptions {
+
+    /**
+     * Participant to put on hold.
+     */
+    private final CommunicationIdentifier targetParticipant;
+
+    /**
+     * Audio to play while on hold.
+     */
+    private final PlaySource playSourceInfo;
+    
+    /**
+     * Operation context.
+     */
+    private String operationContext;
+
+    /**
+     * Create a new HoldOptions object.
+     * @param targetParticipant Participant to be put on hold.
+     * @param playSourceInfo Audio to be played while on hold.
+     */
+    public HoldOptions(CommunicationIdentifier targetParticipant, PlaySource playSourceInfo) {
```

Comment:
`playSourceInfo` is optional. Please remove it from the constructor.

Reply:
changed playSourceInfo  to optional